basiclibrary/lib/src/test/java/basiclibrary/LibraryTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LibraryTest {
+    @Test void someLibraryMethodReturnsTrue() {
+        Library classUnderTest = new Library();
+        assertTrue(classUnderTest.someLibraryMethod(), "someLibraryMethod should return 'true'");
+    }
+
+    @Test void rollReturnsArray() {
+        Library library = new Library();
+        int[] returnedArray;
+        returnedArray = library.roll(5);
+        assertEquals(new int[5], returnedArray);
```

Comment:
Testing whether random values generated correctly is tricky! Other options might be testing that the array is the correct length, or whether any of the values in the array are higher or lower than the maximum/minimum allowed values.